packages/react-scripts/utils/createJestConfig.js

```diff
@@ -21,7 +21,7 @@ module.exports = (resolve, rootDir, isEjecting) => {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     moduleFileExtensions: ['jsx', 'js', 'json'],
     moduleNameMapper: {
-      '^.+\\.(ico|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': resolve('config/jest/FileStub.js'),
+      '^.+\\.(?!(js|jsx|css|json|html)$)[^\\.]+$': resolve('config/jest/FileStub.js'),
```

Comment:
The `.html` extension shouldn't be needed here because importing `.html` files is not supported.